frontend/www/js/omegaup/problem/mine.js

```diff
@@ -0,0 +1,72 @@
+import Vue from 'vue';
+import problem_Mine from '../components/problem/Mine.vue';
+import { OmegaUp, T, API } from '../omegaup.js';
+import UI from '../ui.js';
+
+OmegaUp.on('ready', () => {
+  const payload = JSON.parse(document.getElementById('payload').innerText);
+  let statement = false;
```

Comment:
nit:

```suggestion
  let showAllProblems = false;
```

de dónde salió ese `statement`? O_o

Reply:
Listo, ya lo cambié. En ese momento se me acabaron las ideas para poner nombres a las variables :S

Reply:
oh ya, pensé que había sido un copy paste de otro lado y eso significaría que tendría que ponerle más atención a este cambio.